frontend/src/app/components/settings/settings.component.html

```diff
@@ -0,0 +1,40 @@
+<div style="text-align: center; display: flex; align-items: center; justify-content: center">
+	<mat-card>
+		<mat-card-header>
+			<mat-card-title>Settings</mat-card-title>
+		</mat-card-header>
+		<form [formGroup]="updateUserForm" (submit)="updateUserInfo()">
+			<mat-form-field appearance="fill">
+			  <mat-label>Username</mat-label>
+			  <input matInput formControlName="username">
+			</mat-form-field>
+			<mat-form-field appearance="fill">
+			  <mat-label>Full Name</mat-label>
+			  <input matInput formControlName="fullName">
+			</mat-form-field>
+			<mat-form-field appearance="outline">
+		    <mat-label>Email Address</mat-label>
+		    <input matInput formControlName="email" placeholder="e.g. jdoe@example.com">
+		  </mat-form-field>
+		    <mat-form-field appearance="outline">
+		      <mat-label>Birth Date</mat-label>
+		      <input readonly matInput [matDatepicker]="picker" formControlName="birthDate" (click)="picker.open()">
+		      <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
+		      <mat-datepicker #picker></mat-datepicker>
+		    </mat-form-field>
+			<mat-form-field appearance="fill">
+			  <mat-label>Gender</mat-label>
+			  <mat-select formControlName="gender">
+			    <mat-option *ngFor="let gender of genderOptions" [value]="gender">
+			        {{gender}}
+		        </mat-option>
+		        <mat-option value="">
+		            Prefer Not To Say
+		        </mat-option>
+			  </mat-select>
+			</mat-form-field>
+	    <button mat-flat-button color="primary" style="font-size: x-large; width: 60%; margin: 10px" [disabled]="!updateUserForm.valid">Update</button>	
+			<button mat-stroked-button routerLink="/home" style="width: 60%">home</button>
```

Comment:
as we add more kinds of settings, are you planning to add more cards, so there's one card for user account info, one card for changing password, one card for privacy settings, etc? I think that would look good, but if so we should move the "home" button to be below the cards rather than on one specific card.